### ISSUE TITLE
[Umbrella] evented pleg disabled alpha CIs

### ISSUE THREAD

**Issue by pacoxu** [kubernetes/test-infra#33666]
- ci-kubernetes-node-e2e-containerd-standalone-mode-all-alpha  https://github.com/kubernetes/test-infra/pull/33659
- pull-kubernetes-node-e2e-containerd-standalone-mode-all-alpha https://github.com/kubernetes/test-infra/pull/33642
- ci-kubernetes-e2e-gci-gce-alpha-features https://testgrid.k8s.io/sig-release-master-blocking#gce-cos-master-alpha-features https://github.com/kubernetes/test-infra/pull/31787
- ci-kubernetes-e2e-kind-alpha-features &  ci-kubernetes-e2e-kind-alpha-beta-features https://github.com/kubernetes/test-infra/pull/31767/
- ci-kubernetes-e2e-gce-cos-k8sbeta-alphafeatures https://github.com/kubernetes/test-infra/pull/32349

Remove the FG once the blocking bug is fixed or the FG is removed.

Related:
- https://github.com/kubernetes/test-infra/pull/33463 this will wait until the FG become beta.

**Comment by pacoxu** [comment]
/sig node 
/assign 
This needs to be tracked later.

**Comment by kannon92** [comment]
/triage accepted
/priority backlog

Not sure what we decided on EventedPleg. This is critical for the feature but AFAIK there is no ownership bringing this feature forward.

**Comment by SergeyKanzhelev** [comment]
@pacoxu are you working on it or we should unassign?

**Comment by BenTheElder** [comment]
Maybe update the assignees on https://github.com/kubernetes/enhancements/issues/3386 ?

**Comment by pacoxu** [comment]
> @pacoxu are you working on it or we should unassign?
/unassign 
This is pending on the bugfix PR which are currently pending for review.
- @HirazawaUi and @hshiina have some working PRs.


> Maybe update the assignees on https://github.com/kubernetes/enhancements/issues/3386 ?

cc  @harche, @haircommander, @ruiwen-zhao 

And we may need an approver or more to help get the correct fix merged as soon as possible.

**Comment by harche** [comment]
> * [@HirazawaUi](https://github.com/HirazawaUi) and [@hshiina](https://github.com/hshiina) have some working PRs.

That's correct, the promising fixes are coming from @HirazawaUi and @hshiina. 

Waiting on this for now on that PR https://github.com/kubernetes/kubernetes/pull/129355#issuecomment-2644427236

**Comment by haircommander** [comment]
@HirazawaUi would you like to be assigned this?

**Comment by HirazawaUi** [comment]
Very happy to move it forward.

/assign
